.gitignore

```diff
@@ -1,3 +1,7 @@
 /ve/
 /_build/
 /docs/_build/
+
+.DS_Store
+.idea/
+.vscode/
```

Comment:
You might want to add these to your `~/.gitignore_global`

Reply:
@JayH5 Good point -> should I remove from here?

Reply:
/shrug doesn't really matter. Just that adding a new gitignore entry to _every_ project every time a new editor comes into fashion seems like not the best way to do things 😜